conf/i18n/constants.js

```diff
@@ -1,5 +1,110 @@
+exports.TRANSLATION_FLAGGER_REGEX = /TranslationFlagger.flag\((\s)*\{[^;]+?\}(\s)*\)/g;
+
 exports.DEV_LOCALE = 'en';
 
-exports.BUILD_LOCALES = ['en', 'es', 'fr', 'de', 'it', 'ja'];
+exports.DEFAULT_LOCALES = ['en'];
 
-exports.TRANSLATION_FLAGGER_REGEX = /TranslationFlagger.flag\((\s)*\{[^;]+?\}(\s)*\)/g;
+exports.ALL_LOCALES = {
```

Comment:
I feel like calling this ALL_LOCALES is a little deceiving, since it's a map not an array. Would it be possible to rename this to something that feels slightly more map-like? My brain kept doing a double take because ALL_LANGUAGES is an array but ALL_LOCALES is a map.

Reply:
How about `LANGUAGE_LOCALE_MAP`?

Reply:
does something like LANGUAGES_TO_LOCALES work? or LANGAUGES_TO_ALL_LOCALES?

Reply:
Lets do `LANGUAGES_TO_LOCALES`